rpc/src/optimistically_confirmed_bank_tracker.rs

```diff
@@ -35,7 +35,34 @@ impl OptimisticallyConfirmedBank {
 pub enum BankNotification {
     OptimisticallyConfirmed(Slot),
     Frozen(Arc<Bank>),
-    Root(Arc<Bank>),
+    /// Send the Bank currently rooted and optionally the parent slots in (Slot, Parent Slot) pairs
+    /// The parent slots contains the slot of the bank.
+    Root((Arc<Bank>, Option<Vec<(Slot, Slot)>>)),
```

Comment:
Instead of a tuple I think we should make two separate enums, can be two separate notifications

1. `NewRootBank(Arc<Bank>)`
2. `NewRootedChain(Vec<Slot>)`, the parents here are inferred by the chain, we do not need the tuple

Reply:
The reason for sending the parents pair can can send the parent for the highest slot in the chain.